Modules/xxmodule.c

```diff
@@ -358,31 +358,32 @@ xx_exec(PyObject *m)
 
     /* Finalize the type object including setting type of the new type
      * object; doing it here is required for portability, too. */
-    if (PyType_Ready(&Xxo_Type) < 0)
-        goto fail;
+    if (PyType_Ready(&Xxo_Type) < 0) {
```

Comment:
Booooh! A static type! It seems possible to store it in a module state and get it from the module state in xx_new().

By the way, XxoObject_Check() macro is unused. Maybe just remove it?

Reply:
The point of this module is to show that static types and global variables still work.